test/fail_compilation/fail809.d

```diff
@@ -5,7 +5,7 @@ TEST_OUTPUT:
 fail_compilation/fail809.d(11): Error: scope variable `dg_` may not be returned
 ---
 */
-@safe int delegate() test(lazy int dg)
+int delegate() test(lazy int dg)
```

Comment:
?

Reply:
This patch makes it so that taking the address of a `lazy` parameter in `@safe` code is illegal, so it would have masked the original error.